helpers/deploy/constants.ts

```diff
@@ -0,0 +1,20 @@
+export enum SUPPORTED_NETWORKS {
+  BSCTESTNET = "bsctestnet",
+  BSCMAINNET = "bscmainnet",
+  SEPOLIA = "sepolia",
+  ETHERUEM = "ethereum",
+  HARDHAT = "hardhat",
+}
+
+export const LZ_ENDPOINTS = {
+  ethereum: "0x66A71Dcef29A0fFBDBE3c6a460a3B5BC225Cd675",
+  bsc: "0x3c2269811836af69497E5F486A85D7316753cf62",
+
+  goerli: "0xbfD2135BFfbb0B5378b56643c2Df8a87552Bfa23",
+  sepolia: "0xae92d5aD7583AD66E49A0c67BAd18F6ba52dDDc1",
+  bsctestnet: "0x6Fcb97553D41516Cb228ac03FdC8B9a0a9df04A1",
+  hardhat: "0x6Fcb97553D41516Cb228ac03FdC8B9a0a9df04A1",
+};
```

Comment:
We are not using this constant in this project. I would remove it

Reply:
Sure, it came from checking out the file from PR 21, we can add it if it gets merged with PR 21